kaisen/kaisen/src/taiwo/api_call.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+note1 = "6.5095° N, 3.3711° E"
+note2 = "https://api.open-meteo.com/v1/forecast?latitude={latitude}&longitude={longitude}&hourly=temperature_2m"
+note3 = "https://api.open-meteo.com/v1/forecast?latitude=52.52&longitude=13.41&current=temperature_2m,wind_speed_10m&hourly=temperature_2m,relative_humidity_2m,wind_speed_10m"
+note4 = "https://api.open-meteo.com/v1/forecast?latitude=6.5095&longitude=3.3711&hourly=temperature_2m,wind_speed_10m&hourly=temperature_2m,relative_humidity_2m,wind_speed_10m"
```

Comment:
I think we would have to give better variable names for them.

Reply:
okay